src/ServicePulse.Host/app/modules/monitoring/js/services/services.monitoring.js

```diff
@@ -62,9 +62,16 @@
                 });
         }
 
+        function checkConnections() {
```

Comment:
This call doesn't "check the connections", the connection status is inferred from the result of getting the list of monitored endpoints. Looking at this file in isolation the name of this method doesn't make sense.

```suggestion
        function getMonitoredEndpoints() {
```